ruleset/rules/0750-github_rules.xml

```diff
@@ -0,0 +1,3225 @@
+        <description>GitHub Dependabot alerts enable.</description>
+        <options>no_full_log</options>
+        <group>git_dependabot_alerts</group>
+    </rule>
+
+    <!-- dependabot_alerts_new_repos
+            disable
+            enable
+     -->
+    <rule id="91117" level="3">
+        <if_sid>91100</if_sid>
+        <action type="osregex">^dependabot_alerts_new_repos.\.*</action>
+        <description>GitHub Dependabot alerts new repos.</description>
+        <options>no_full_log</options>
+        <group>git_dependabot_alerts_new_repos</group>
+    </rule>
+
+    <!-- dependabot_alerts_new_repos.disable -->
+    <rule id="91118" level="9">
+        <if_sid>91117</if_sid>
+        <action>dependabot_alerts_new_repos.disable</action>
+        <description>GitHub Dependabot alerts new repos disable.</description>
+        <options>no_full_log</options>
+        <group>git_dependabot_alerts_new_repos</group>
+    </rule>
+
+    <!-- dependabot_alerts_new_repos.enable -->
+    <rule id="91119" level="5">
+        <if_sid>91117</if_sid>
+        <action>dependabot_alerts_new_repos.enable</action>
+        <description>GitHub Dependabot alerts new repos enable.</description>
+        <options>no_full_log</options>
+        <group>git_dependabot_alerts_new_repos</group>
+    </rule>
+
+    <!-- dependabot_security_updates
+            disable
+            enable
+    -->
+    <rule id="91120" level="3">
+        <if_sid>91100</if_sid>
+        <action type="osregex">^dependabot_security_updates.\.*</action>
+        <description>GitHub Dependabot security updates.</description>
+        <options>no_full_log</options>
+        <group>git_dependabot_security_updates</group>
+    </rule>
+
+    <!-- dependabot_security_updates.disable -->
+    <rule id="91121" level="9">
+        <if_sid>91120</if_sid>
+        <action>dependabot_security_updates.disable</action>
+        <description>GitHub Dependabot security updates disable.</description>
+        <options>no_full_log</options>
+        <group>git_dependabot_security_updates</group>
+    </rule>
+
+    <!-- dependabot_security_updates.enable -->
+    <rule id="91122" level="5">
+        <if_sid>91120</if_sid>
+        <action>dependabot_security_updates.enable</action>
+        <description>GitHub Dependabot security updates enable.</description>
+        <options>no_full_log</options>
+        <group>git_dependabot_security_updates</group>
+    </rule>
+
+    <!-- dependabot_security_updates_new_repos
+            disable
+            enable
+    -->
+    <rule id="91123" level="3">
+        <if_sid>91100</if_sid>
+        <action type="osregex">^dependabot_security_updates_new_repos.\.*</action>
+        <description>GitHub Dependabot security updates new repos.</description>
+        <options>no_full_log</options>
+        <group>git_dependabot_security_updates_new_repos</group>
+    </rule>
+
+    <!-- dependabot_security_updates_new_repos.disable -->
+    <rule id="91124" level="9">
```

Comment:
level 9 is "Error from invalid source", perhaps level 12 "High importance event" is more appropriate

Reply:
Done